src/Illuminate/Database/Connection.php

```diff
@@ -1035,9 +1048,7 @@ public function registerDoctrineType(string $class, string $name, string $type):
             Type::addType($name, $class);
         }
 
-        $this->getDoctrineSchemaManager()
-            ->getDatabasePlatform()
-            ->registerDoctrineTypeMapping($type, $name);
+        $this->doctrineTypeMappings[$name] = $type;
```

Comment:
I've kept this whole `registerDoctrineType` method for backwards compatibility but it can be removed in Laravel 9 because the same method on the DatabaseManager class replaces it. I'll PR that today too.